docs/management/connectors/action-types/email.asciidoc

```diff
@@ -229,3 +246,58 @@ secrets:
 <2> Many organizations support use of your email address as your username.
     Check with your system administrator if you receive
     authentication-related failures.
+
+Before you create an email connector for Microsoft Exchange, you must create and register the client integration application on the https://go.microsoft.com/fwlink/?linkid=2083908[Azure portal]:
```

Comment:
I think we need a section header here to identify that this is how to set up the new configuration for MS Exchange. I was confused when I first read through because it talked about the deprecated config but then moved on to the new config without a transition.